code/game/gamemodes/blob/powers.dm

```diff
@@ -350,39 +350,39 @@
 			BS.Goto(pick(surrounding_turfs), BS.move_to_delay)
 	return
 
-
 /mob/camera/blob/verb/split_consciousness()
 	set category = "Blob"
 	set name = "Split consciousness (100) (One use)"
 	set desc = "Expend resources to attempt to produce another sentient overmind."
 
+	var/turf/T = get_turf(src)
+	if(!T)
+		return
 	if(split_used)
 		to_chat(src, "<span class='warning'>You have already produced an offspring.</span>")
 		return
 	if(is_offspring)
 		to_chat(src, "<span class='warning'>You cannot split as an offspring of another Blob.</span>")
 		return
-	if(!blob_nodes || !blob_nodes.len)
-		to_chat(src, "<span class='warning'>A node is required to birth your offspring...</span>")
-		return
-	var/obj/structure/blob/node/N = locate(/obj/structure/blob) in blob_nodes
+
+	var/obj/structure/blob/N = (locate(/obj/structure/blob) in T)
 	if(!N)
-		to_chat(src, "<span class='warning'>A node is required to birth your offspring...</span>")
+		to_chat(src, "<span class='warning'>A node is required to birth your offspring..</span>")
```

Comment:
Why is the ellipsis reduced from three dots to two? It makes more sense to either leave it at 3, or convert it to a period/exclamation mark. imo a period is better here, anyways.

Reply:
Fixed.